xwiki-platform-core/xwiki-platform-wysiwyg/xwiki-platform-wysiwyg-client/src/main/java/org/xwiki/gwt/wysiwyg/client/gadget/SelectGadgetWizardStep.java

```diff
@@ -73,8 +73,7 @@ public Object getResult()
         GadgetInstance gadgetInstance = new GadgetInstance();
         gadgetInstance.setMacroCall(superResult);
         // prefill the title of the gadget with the name of the macro
-        gadgetInstance.setTitle(superResult.getName());
-
+        gadgetInstance.setTitle($services.localization.render('rendering.macro.' + superResult.getName()));
```

Comment:
Missing quotes and pretty much the stuff that was in the previous commit. Remember you`re in java here and the `setTitle` method accepts a String as a parameter and you are building that string.